Ship_Game/Ships/Ship.cs

```diff
@@ -436,9 +436,9 @@ float GetYBankAmount(FixedSimTime timeStep)
             }
         }
 
-        public bool IsPlatformOrStation => ShipData.IsPlatformOrStation;
-        public bool IsShipyard => ShipData.IsShipyard;
-        public bool IsStation => ShipData.IsStation;
+        public bool IsPlatformOrStation => ShipData.IsPlatformOrStation || MaxFTLSpeed == 0 || MaxSTLSpeed == 0 ;
+        public bool IsShipyard => ShipData.IsShipyard || MaxFTLSpeed == 0 || MaxSTLSpeed == 0;
+        public bool IsStation => ShipData.IsStation || MaxFTLSpeed == 0 || MaxSTLSpeed == 0;

```

Comment:
This is not needed. In game all of this is ok without the speed. 
The only place it needs to be checked is the CreateStartingShips() edge case there a prototype might be a station and therefore its roledata is prototype.